packages/hoprd/src/admin.ts

```diff
@@ -38,6 +38,14 @@ export class AdminServer {
       return false
     }
 
+    if (req.url) {
```

Comment:
Why is this needed? A one-liner reasoning would be useful.

Reply:
Other clients different to `hoprd` might pass the `apiToken` via a query param since they won't be on the same domain the node is hosted. This detects these cases and provides the ability for any client that knows the `apiToken` to reach your HOPR node.

Adding it as a comment in the code.